CHANGES.md

```diff
@@ -10,6 +10,8 @@ This document is intended for Spotless developers.
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `1.27.0`).
 
 ## [Unreleased]
+### Changes
+* We are now opting in to Gradle's new stable configuration cache. ([#1591](https://github.com/diffplug/spotless/pull/1591))
```

Comment:
This is only a behavior of Spotless self, users still need to enable stable CC flags in their projects if they want. 
And next step, we might need to add `--configuration-cache` flags in tests to test compatibitlies.

Reply:
Thanks! This changelog is intended mostly for Spotless developers, so I usually put important changes to the build into it. The gradle and maven plugins have their own changelogs for end-users.